apps/backoffice-v2/src/lib/blocks/hooks/useUbosUserProvidedBlock/useUbosUserProvidedBlock.tsx

```diff
@@ -1,8 +1,47 @@
 import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
-import { omitPropsFromObject } from '@/pages/Entity/hooks/useEntityLogic/utils';
+import { TextWithNAFallback } from '@ballerine/ui';
+import { createColumnHelper } from '@tanstack/react-table';
 import { useMemo } from 'react';
 
-export const useUbosUserProvidedBlock = ubosUserProvided => {
+export const useUbosUserProvidedBlock = (
```

Comment:
Boyscout. Added types and now using columnHelper instead of an array of objects for columns.